### PR TITLE
19261 continuation in - initial changes

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#19261

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/src/legal_api/models/business.py [hunk 1]
```diff
@@ -156,6 +169,10 @@ class AssociationTypes(Enum):
             'numberedDescription': 'Numbered Community Contribution Company'
         }
     }
+    BUSINESSES[LegalTypes.BCOMP_CONTINUE_IN] = BUSINESSES[LegalTypes.BCOMP]
+    BUSINESSES[LegalTypes.CONTINUE_IN] = BUSINESSES[LegalTypes.COMP]
+    BUSINESSES[LegalTypes.ULC_CONTINUE_IN] = BUSINESSES[LegalTypes.BC_ULC_COMPANY]
+    BUSINESSES[LegalTypes.CCC_CONTINUE_IN] = BUSINESSES[LegalTypes.BC_CCC]
```
Comment (severinbeauvais) [inline legal-api/src/legal_api/models/business.py:175]: How about a code comment explaining what/why you're doing this here?

File: legal-api/tests/unit/resources/v2/test_business.py [hunk 2]
```diff
@@ -377,31 +379,34 @@ def test_draft_amalgamation_name_selection(session, client, jwt, identifier, leg
...
     """Assert that the affiliated businesses endpoint returns as expected."""
     # setup
-    identifiers = ['CP1234567', 'BC1234567', 'Tb31yQIuBw', 'Tb31yQIuBq', 'Tb31yQIuBz', 'Tb31yQIuBy']
+    identifiers = ['CP1234567', 'BC1234567', 'Tb31yQIuBv', 'Tb31yQIuBw', 'Tb31yQIuBx', 'Tb31yQIuBy', 'Tb31yQIuBz']
     businesses = [
         (identifiers[0], Business.LegalTypes.COOP.value, None),
         (identifiers[1], Business.LegalTypes.BCOMP.value, '123456789BC0001')]
     draft_businesses = [
-        (identifiers[2], 'registration', Business.LegalTypes.BCOMP.value, None),
-        (identifiers[3], 'incorporationApplication', Business.LegalTypes.SOLE_PROP.value, 'NR 1234567'),
```
Comment (severinbeauvais) [inline legal-api/tests/unit/resources/v2/test_business.py:400]: Yes, these 2 lines are wrong :)

**Review by severinbeauvais — APPROVED** [review]
Thanks for all the tests, cleanup and bugfixes. LGTM.

**Review by jamespaologarcia — APPROVED** [review]
LGTM

**Review by ketaki-deodhar — APPROVED** [review]

**Pull request merged**